synapseAnnotations/data/common/minimal_Sage_standard.json

```diff
@@ -1266,29 +1266,29 @@
   }, 
   {
     "name": "species", 
-    "description": "", 
+    "description": "The name of a species (typically a taxonomic group) of organism.", 
     "columnType": "STRING", 
     "maximumSize": 250, 
     "enumValues": [
       {
         "value": "Human", 
-        "description": "", 
-        "source": ""
+        "description": "Homo sapiens", 
+        "source": "https://meshb.nlm.nih.gov/record/ui?ui=D006801"
       }, 
       {
         "value": "Mouse", 
-        "description": "", 
-        "source": ""
+        "description": "The common name for the genus Mus", 
+        "source": "https://meshb.nlm.nih.gov/record/ui?ui=D051379"
```

Comment:
Making a note for the record that while technically correct, this is not specific (and why I advocated for NCBI TAXON IDs), also for Rat below.